pkg/refactor/database/db_schema.sql

```diff
@@ -90,6 +90,33 @@ CREATE TABLE IF NOT EXISTS "secrets" (
     FOREIGN KEY ("namespace_id") REFERENCES "namespaces"("oid") ON DELETE CASCADE ON UPDATE CASCADE
 );
 
+DROP TABLE IF EXISTS "runtime_variables";
+CREATE TABLE IF NOT EXISTS "runtime_variables" (
+    "id" uuid,
+    "namespace_id" uuid,
+    "workflow_id" uuid,
+    "instance_id" uuid,
+
+    "scope" text NOT NULL,
```

Comment:
What value does the "scope" field serve? It can be derived by looking at which uuid fields are not null.

Reply:
Good idea. Fixed :)